stock_picking_back2draft/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Pickings back to draft",
+    "summary": "Reopen cancelled pickings",
+    "version": "11.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://www.agilebg.com",
```

Comment:
@aheficent Could you change website to OCA repo ?

Reply:
Sure, done now. Thanks for reviewing.